srcpkgs/NetworkManager/template

```diff
@@ -35,18 +35,11 @@ maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Projects/NetworkManager"
 distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=2b29ccc1531ba7ebba95a97f40c22b963838e8b6833745efe8e6fb71fd8fca77
+checksum=0c8e80e77877860e4a4e6ab4a0f7cdc1186e356b65b042a751897188b88944d2
 # TODO: Some tests require network namespaces to run.
 make_check=extended
 lib32disabled=yes
 conf_files="/etc/${pkgname}/${pkgname}.conf"
-make_dirs="
- /etc/NetworkManager/system-connections 0755 root root
- /etc/NetworkManager/dispatcher.d/pre-up.d 0750 root root
- /etc/NetworkManager/dispatcher.d/pre-down.d 0750 root root
- /etc/NetworkManager/VPN 0755 root root
- /var/lib/NetworkManager 0755 root root
-"
```

Comment:
Why remove these?

Reply:
The build output states that those directories get removed during the process of creating the package, so I thought they weren't necessary.

Reply:
They're removed because they're empty. We need to create them again at installation.

Reply:
I was under the impression that `make_dirs` only created directories within the package, not at installation time? Or did I confuse that with another option?

Reply:
Just checked the manual, you were right, I'll fix this in my next push.